### DISCUSSION TITLE
Models with multiple animations do not work in Scene Viewer

### DISCUSSION THREAD

**Discussion by jangarita** [google/model-viewer#3643] [Q&A]
### Description
When you try to show a model with multiple animations using Scene Viewer it crashes or is not visible.
Works well using WebXR but not using Scene Viewer.

#### Live Demo
Scene Viewer version (without https):
http://labs.jangarita.com/svbug/

WebXR version (with https):
https://labs.jangarita.com/svbug/

### Version
model-viewer: v1.3.0

### Browser Affected
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS
- [x] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

**Comment by elalish** [comment]
@tpsiaki for the scene-viewer bug. Keep in mind that animation selection will not work in scene-viewer regardless, as we don't have a way to tell it which animation to use.
